addons/test_mail_full/tests/test_sms_performance.py

```diff
@@ -66,7 +66,7 @@ def test_message_sms_record_1_partner(self):
     def test_message_sms_record_10_partners(self):
         record = self.test_record.with_user(self.env.user)
         pids = self.partners.ids
-        with self.mockSMSGateway(), self.assertQueryCount(employee=41):  # test_mail_enterprise: 43
+        with self.mockSMSGateway(), self.assertQueryCount(employee=51):  # test_mail_enterprise: 43
```

Comment:
What are the 10 extra queries in this case? Do you have the exact list?
This seems like a bit too much.